lte/gateway/c/oai/lib/3gpp/3gpp_24.301.h

```diff
@@ -196,18 +198,58 @@ typedef struct ue_network_capability_s {
   /* NF notification procedure capability */
 #define UE_NETWORK_CAPABILITY_NF 1
   uint8_t nf : 1;
-  // uint8_t spare[0..8]
+
+  /* Bits 8 to 6 of octet 8 are spare and shall be coded as zero */
+  /* S1-U data transfer enabled */
+  uint8_t epco : 1;
+  uint8_t hccpciot : 1;
+  uint8_t erwfopdn : 1;
+#define UE_NETWORK_CAPABILITY_S1UDATA 1
+  uint8_t s1udata : 1;
+  /* User plane CIoT EPS Optimization */
+#define UE_NETWORK_CAPABILITY_UPCIOT 1
+  uint8_t upciot : 1;
+  /* Control plane CIoT EPS Optimization */
+#define UE_NETWORK_CAPABILITY_CPCIOT 1
+  uint8_t cpciot : 1;
+  /* ProSe UE-to-network-relay */
+#define UE_NETWORK_CAPABILITY_PROSERELAY 1
+  uint8_t proserelay : 1;
+  /* ProSe direct communication */
+#define UE_NETWORK_CAPABILITY_PROSEDC 1
+  uint8_t prosedc : 1;
+
+  /* Bits 8 to 6 of octet 9 are spare and shall be coded as zero */
```

Comment:
Same here, no spare bits in this octet either

Reply:
Removed